### DISCUSSION TITLE
请求添加京东云亚瑟的固件

### DISCUSSION THREAD

**Discussion by daiyou519** [immortalwrt/immortalwrt#1315] [General]
请求添加京东云亚瑟（ipq60xx-jdc_ax1800-pro）的固件。

**Comment by yjzmh** [comment]
同求

**Comment by 1715173329** [comment]
推荐使用 coolsnowwolf 的仓库。

**Reply by jwkinko** [reply]
coolsnowwolf仓库 ipq6000 固件有个问题，重启必须断电才能启动

**Comment by KardRi** [comment]
我試了一下，大概改這些東西，就有機會能用
target/linux/qualcommax
package/boot/uboot-envtools/files/qualcommax_ipq60xx
package/firmware/ipq-wifi
package/kernel/linux/modules/netfilter.mk
package/kernel/nat46
package/kernel/qca-nss-dp
package/kernel/qca-ssdk

那一坨qca-的kmod，如果要用，把feeds.conf.default也照抄，從qosmio那邊拉

其他都不用改
![image](https://github.com/immortalwrt/immortalwrt/assets/51556723/6162c2ed-a3f0-4f8e-937b-27999e0fb8ff)
![image](https://github.com/immortalwrt/immortalwrt/assets/51556723/5dc4e9db-f87e-454c-b254-a214b7e78d2b)

**Reply by gdanmac** [reply]
能提供教程吗没明白，或者提供一个固件

**Reply by KardRi** [reply]
你去找我主頁，後來我是直接用其他repo的固件，他的package/luci是用immortalwrt的，大概只有kernel patch和firmware patch不一樣(?)

**Reply by gdanmac** [reply]
感谢

**Comment by VIKINGYFY** [comment]
https://github.com/VIKINGYFY/immortalwrt.git

我搞了一个，有兴趣可以直接用

**Reply by zz1905501566** [reply]
这个咋用

**Comment by zz1905501566** [comment]
请求官方更新京东云亚瑟ipq6000官方immortalwrt固件

**Comment by jwkinko** [comment]
请求添加ax5京东云版

**Comment by lpflhh** [comment]
请求添加京东云亚瑟（ipq60xx-jdc_ax1800-pro）的固件，我们群里有不下两百人有这台机器，想用官方immortalwrt固件，但是在make menuconfig 型号选择菜单里没有找到这台设备的名字 jdc_ax1800-pro

**Comment by alansdragon0** [comment]
同求，这个型号的存量不少的，希望后续能支持，谢谢作者大神！

**Comment by hoicau** [comment]
[openwrt/openwrt](https://github.com/openwrt/openwrt) 的 main 分支已经有支持了，firmware selector 里选择 snapshot 检索型号 `JDCloud RE-SS-01` 即可。~~不知道这边什么时候会 merge~~实际上已经 merge 了，见 [ipq60xx.mk#L103](https://github.com/immortalwrt/immortalwrt/blob/b8c93e2c66af061b839e92eb3fb2bf366a929f1c/target/linux/qualcommax/image/ipq60xx.mk#L103)。

**Reply by hoicau** [reply]
刷写方法（中文）：

1. 准备镜像： 下载 initramfs 镜像文件，将其重命名为 initramfs.itb，并放置在 TFTP 服务器的根目录下。

2. 中断启动： 如果原厂固件版本为 4.3.0.r4211 或更高，当屏幕出现 “disabled console and autoboot in 2 seconds” 提示时，输入 jdmt018R 以中断 U-Boot 自动启动。

3. 运行命令： 在 U-Boot 控制台中依次执行以下命令：

```
tftpboot initramfs.itb

bootm
```

4. 正式升级： 等待 OpenWrt 临时系统启动后，通过 SCP 工具或 LuCI 网页后台上传 sysupgrade.bin 固件包，完成最终的系统升级。

<details>

<summary>英文原文</summary>

https://git.openwrt.org/?p=openwrt/openwrt.git;a=commit;h=16422510b52af3f14d126923968d34373ef6c7b7

Flash instructions:
  1. Download the initramfs image, rename it to
     initramfs.itb, host it with the tftp server.
  2. If the stock firmware version is 4.3.0.r4211
     or higher, enter "jdmt018R" to interrupt
     U-Boot when the following statement appears:
     "disabled console and autoboot in 2 seconds"
  3. Run these commands in U-Boot console:
     tftpboot initramfs.itb
     bootm
  4. After openwrt boots up, use scp or luci web
     to upload sysupgrade.bin to upgrade.

</details>

**Reply by hoicau** [reply]
建议 feeds 加 nss-packages 和 sqm_scripts_nss自行编译

**Comment by chanfa7** [comment]
请求添加京东云亚瑟（ipq60xx-jdc_ax1800-pro）的固件

**Reply by Namis0** [reply]
> 请求添加京东云亚瑟（ipq60xx-jdc_ax1800-pro）的固件
